CHANGELOG.md

```diff
@@ -18,10 +18,26 @@
 -->
 
 # Table of Contents
-
+- [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+# 2.0.0
+
+This is release candidate.
+
+# Core
+
+- Refactor front-end with admin-api.
+- Manage-api removes dependency on mysql.
```

Comment:
`mysql` -> `MySQL`

Reply:
manager-api or Manger-API, I would prefer the first one.